src/pe_reports/report_generator.py

```diff
@@ -22,12 +39,28 @@ def load_template():
 
 def export_set(prs):
     """Export PowerPoint report set to output directory."""
-    pptx_out = "Customer_ID_Posture_Exposure.pptx"
-    prs.save(os.path.join(SAVE_PATH, pptx_out))
+    try:
+        pptx_out = "Customer_ID_Posture_Exposure.pptx"
+        prs.save(os.path.join("/output", pptx_out))
+    except OSError:
+        print("No output available.")
```

Comment:
Can this be modified so that the exception output is also printed here?  `No output available` seems a bit vague to me.

Reply:
Resolved in 4c928bd

Reply:
I don't think commit 4c928bd added the exception output to this particular case.

Reply:
@jsf9k yes that was a mistake. It is now resolved with commit - 882be6a